### ISSUE TITLE
finalize tree ID info to submit with permit

### ISSUE THREAD

**Issue by lizzieinvancouver** [cchambe12/treespotters#2]
@christophe-rd I am ready to submit the proposal to the Arb but I really think your tree IDs are NOT the ones the Arboretum uses.  As @cchambe12 wrote on 27 Oct 2024 (thanks @cchambe12):

> Next, I was able to track down the IDs that the Arboretum uses through this portal (https://arboretum.harvard.edu/explorer/). They are actually using column AA in the status_observation_data.xlsx sheet when you download this zip file (https://arboretum.harvard.edu/wp-content/uploads/2020/12/TreeSpottersData_2015-2020.zip), which is in this sort of format: 187-2006*B, Tree 1. You can find a bit more information on the zipped file on the Data Resources (https://arboretum.harvard.edu/research/data-resources/) page on the Arboretum website. 

When I go to the portal, every tree I look at has an ID with a hypen and letters and numbers ... so I don't think the treeswithPhenodata.csv is the IDs we want (as they are all 6 digit numerical). I think you need to transcribe from those IDs to the ones in the explorer portal. Can you work on this and send me an updated list so I can send off our proposal? 

Thanks!

**Comment by christophe-rd** [comment]
@lizzieinvancouver sorry! I should've read Cat's email more carefully. I confused `Individual_ID` (e.g. 87761) column with `Plant_nickname` (e.g. 358-82*A, Tree 8). Anyways, I printed them both along with the species. 

Sorry for not adding it to the repo. The data file I used is >130mb and git won't let me push it, so until I figure how to fix this, I joined the file here:

[treeswithPhenodata.csv](https://github.com/user-attachments/files/17970434/treeswithPhenodata.csv)
